docs/mmteb/points.md

```diff
@@ -3,7 +3,7 @@
 | GitHub            | Total points | New dataset | New task | Dataset annotations | (Bug)fixes | Running Models | Review PR |  Paper Writing | Ideation | Coordination |
 |-------------------| ------------ |-------------| -------- | ------------------- | ---------- | -------------- |-----------| -------------- | -------- | ------------- |
 | KennethEnevoldsen |              | 54          |          |                   8 |         16 |                | 20        |                |          |             5 |
```

Comment:
Will you add a point for the review here as well

Reply:
Yeah, will do!